client/libs/material/src/lib/sermon-table/sermon-table.component.scss

```diff
@@ -0,0 +1,47 @@
+@import 'libs/theme/variables.scss';
+@import 'libs/theme/main.scss';
+
+.full-width-table {
+  width: 100%;
+}
+
+.bg-color-primary {
+  background-color: $primary;
+}
+
+.create-btn {
+  min-width: 40px !important;
+}
+
+.create-icon {
+  color: $accent;
```

Comment:
we have global text color classes in `_fonts.scss`.
This comment is for all other cases you are using `color: $some-var`